tests/ng2web/src/app/shared/sdk/resolvers/auth-account.ts

```diff
@@ -0,0 +1,19 @@
+/* tslint:disable */
+import { Observable } from 'rxjs/Observable';
+import { Injectable } from '@angular/core';
+import { Resolve, ActivatedRouteSnapshot } from '@angular/router';
+import { Store } from '@ngrx/store';
+
+import { User } from '../models/user';
```

Comment:
This is generated depending on the api configuration.

The test api is using Account.